pxr/base/arch/defines.h

```diff
@@ -35,15 +35,17 @@
 #if defined(i386) || defined(__i386__) || defined(__x86_64__) || \
     defined(_M_IX86) || defined(_M_X64)
 #define ARCH_CPU_INTEL
-#elif defined(__arm__) || defined(__aarch64__) || defined(_M_ARM)
+#elif defined(__arm__) || defined(__aarch64__) || defined(_M_ARM) || \
+    defined(_M_ARM64)
```

Comment:
Along the lines of @dgovil's comment in `build_usd.py`, I think we'd also want to include `defined(_M_ARM64EC)` here and below?